lecture-notes/2-vcs.md

```diff
@@ -1673,10 +1673,94 @@ TODO: Talk about interactive rebasing for squashing/prettifying.
 
 CONTENTS: Collaboration with Git
 
+You've done enough solo development. It's now time to start working with your
+groupmates on your project. In order to do that, you need to make your
```

Comment:
Does "your project" refer to some abstract project, or is it a reference to the example (calculator) repository that earlier notes have been using as an example?

Reply:
abstract. if you have a 1-2 sentence intro that i can use to explain that i would be grateful

Reply:
cc @tchebb

Reply:
What do you feel is missing? I think this is fine TBH.